### PR TITLE
sheryl_stillman_pr_school_challenge_2008

### PR DESCRIPTION
**Pull request by stillsheryl**

### CODE REVIEW

File: lib/school.rb [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    @hours_in_school_day = hours_in_school_day
+    @student_names = []
+  end
+
+  def add_student_name(student)
+    student_names << student
+  end
+
+  def end_time
+    return "#{@start_time.to_i + @hours_in_school_day}" + ":00"
```
Comment (mikedao) [inline lib/school.rb:14]: Good approach. You don't need the explicit return here.

File: lib/school.rb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  end
+
+  def is_full_time?
+    true if @hours_in_school_day >= 4
```
Comment (mikedao) [inline lib/school.rb:18]: this returns nil if it isn't full time, Would rather have an if/else statement that returns false.

Comment (stillsheryl) [reply lib/school.rb:18]: Thank you, Mike! I had it as an if/else at first but then changed it. Good to know that it's better the other way!

Comment (stillsheryl) [reply lib/school.rb:18]: def is_full_time?
  @hours_in_school_day >= 4
end

**Pull request closed without merging**